src/familiar/itemFamiliar.ts

```diff
@@ -0,0 +1,66 @@
+import { Familiar, myFamiliar, runChoice, useFamiliar, visitUrl } from "kolmafia";
+import { $familiar, $item, findFairyMultiplier, get, have, set } from "libram";
+import { menu } from "./freeFightFamiliar";
+
+let bestNonCheerleaderFairy: Familiar;
+
+export function bestFairy(): Familiar {
+  if (have($familiar`Trick-or-Treating Tot`) && have($item`li'l ninja costume`)) {
+    return $familiar`Trick-or-Treating Tot`;
+  }
+
+  if (
+    have($familiar`Steam-Powered Cheerleader`) &&
+    findFairyMultiplier($familiar`Steam-Powered Cheerleader`) > 1.25
+  ) {
+    return $familiar`Steam-Powered Cheerleader`;
+  }
```

Comment:
If we are specifically handling cheerleader, should we also separately handle mutant fire ant?

Reply:
No; cheerleader changes over the course of the day, which is bad with caching. fire ant changes over the course of the month/week/whatever, and thus can be cached properly just fine and dandy.

That being said, we could add handling to make sure the cheerleader is in fact the best fairy.

Reply:
I thought that the multipliers were wonky (read: different from a weight multiplier) for mutant familiars, which was a massive headache in the handling in yachtzee. 
It appears that we do not in fact use multipliers for calculations in yachtzee anymore specifically for this reason. I'm not sure if this would be properly captured here (or for meatFamiliar for that matter)

Reply:
[here](https://github.com/Loathing-Associates-Scripting-Society/libram/blob/d203a4bc24ea6752de3934073b16263ccfb395aa/src/lib.ts#L679-L687) is where findFairyMultiplier is defined; while you're right that the efficacy multiplier is different from the weight multiplier, but I still think treating them as more-or-less the same is "basically" fine